### PR TITLE
ISS-924 Add Dependency FAQ

### PR DESCRIPTION
**Pull request by dhafley**
/close #work

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -39,3 +39,11 @@ See our [Preferences](configuring.md#configuring-automatic-formatting) page for
 ### Where can I learn more and discuss Pixeebot?
 
 Users can join the Pixee community [on Slack](https://join.slack.com/t/openpixee/shared_invite/zt-1pnk7jqdd-kfwilrfG7Ov4M8rorfOnUA). This channel can be used to engage with peers who are also interested in Pixee. Feel free to email us at help@pixee.ai with any questions or comments.
+
+### Why does pixee sometimes add new dependencies to my project?
+
+We always prefer to use existing controls built into a language, or a control from a well-known and trusted community dependency. When this is not an option, we add our own open source dependency to the project to ensure maximum readability and maintainability. All dependencies utilize a MIT License, which permits anyone to view, use, modify, distribute, and contribute to the MIT-licensed code.
```
Comment (drdavella) [inline docs/faqs.md:45]: Our own security packages are MIT but this is not strictly true as Python adds some third-party packages with other permissive licenses. 
```suggestion
We always prefer to use existing controls built into a language, or a control from a well-known and trusted community dependency. When this is not an option, we add our own open source dependency to the project to ensure maximum readability and maintainability. All dependencies utilize permissive open-source licenses.
```

**Review by drdavella — CHANGES_REQUESTED** [review]

**Review by drdavella — APPROVED** [review]

File: docs/faqs.md [hunk 2]
```diff
@@ -39,3 +39,11 @@ See our [Preferences](configuring.md#configuring-automatic-formatting) page for
 ### Where can I learn more and discuss Pixeebot?
 
 Users can join the Pixee community [on Slack](https://join.slack.com/t/openpixee/shared_invite/zt-1pnk7jqdd-kfwilrfG7Ov4M8rorfOnUA). This channel can be used to engage with peers who are also interested in Pixee. Feel free to email us at help@pixee.ai with any questions or comments.
+
+### Why does pixee sometimes add new dependencies to my project?
```
Comment (gilday) [inline docs/faqs.md:43]: Uppercase Pixee for consistency

File: docs/faqs.md [hunk 3]
```diff
@@ -39,3 +39,11 @@ See our [Preferences](configuring.md#configuring-automatic-formatting) page for
...
+
+We always prefer to use existing controls built into a language, or a control from a well-known and trusted community dependency. When this is not an option, we add our own open source dependency to the project to ensure maximum readability and maintainability. All dependencies utilize permissive open-source licenses.
```
Comment (gilday) [inline docs/faqs.md:45]: Consider expanding on "when this is not an option". Why is not an option? Maybe "when no such dependency exists that meets that criteria"

File: docs/faqs.md [hunk 4]
```diff
@@ -39,3 +39,11 @@ See our [Preferences](configuring.md#configuring-automatic-formatting) page for
...
+
+Learn more about the [Java Security Toolkit (io.github.pixee.java-security-toolkit) on Maven Central](https://central.sonatype.com/artifact/io.github.pixee/java-security-toolkit/overview).
```
Comment (gilday) [inline docs/faqs.md:47]: There's another Java toolkit we maintain specifically for xstream.

**Review by gilday — APPROVED** [review]

**Pull request merged**